cmd/mirroring/logInfo.txt

```diff
@@ -0,0 +1,32 @@
+1528952 e0a8822e61ba905c6efb216666c3e09f37f403c5d1f55e8cef8d964ddddefabf
```

Comment:
can you remove tihs file from the PR?

Reply:
and the .DS_Store's above. These are apple thumbnails that shouldn't be checked into the repo